test/gtest/common/test_helpers.cc

```diff
@@ -24,7 +24,7 @@ namespace ucs {
 
 typedef std::pair<std::string, ::testing::TimeInMillis> test_result_t;
 
-const double test_timeout_in_sec = 60.;
+const double test_timeout_in_sec = 20.;
```

Comment:
instead, update get_deadline() so it will have upper limit that is derived from watchdog_timeout.
for example 50% or 75% of watchdog_timeout